lib/pf/Authentication/Source/HTTPSource.pm

```diff
@@ -30,6 +30,7 @@ has 'username' => ( isa => 'Maybe[Str]', is => 'rw', default => undef );
 has 'password' => ( isa => 'Maybe[Str]', is => 'rw', default => undef );
 has 'authentication_url' => ( isa => 'Str', is => 'rw', default => '' );
 has 'authorization_url' => ( isa => 'Str', is => 'rw', default => '' );
+has 'realm' => (isa => 'ArrayRef[Str]', is => 'rw');
```

Comment:
Since that field seems common to all sources, it should be moved in Authentication/Source.pm